kevinb5617/ResuMake
# Added initial logo designs

# Description

I added two `.png` files that serve as the main "title" logo and a small version, which can be used as a favicon.

# Related Issue(s) and Pull Request(s)

Addresses #6

# Solution

I used PowerPoint to create the logos and an online tool to make the png's transparent.

# Steps to Test

1. `git fetch upstream`
2. `git checkout feature/initial-logo-design`
3. `cd images`
4. Open the two images files in your favorite image viewer to see them

# Known Issues

* There are no glaring, issues, but the quality of the logo may not be amazing in a large canvas.

# Checklist
